tests/conftest.py

```diff
@@ -475,6 +513,70 @@ def sample_static_architecture():
     }
 
 
+@pytest.fixture
+def sample_quality_metric_set():
+    return {
+        'calibration_set_id': UUID('e70667f9-a432-4585-97a9-d54de9a85abd'),
```

Comment:
```suggestion
        'calibration_set_id': 'e70667f9-a432-4585-97a9-d54de9a85abd',
```
This dict represents the raw JSON response, right?